shared/studio/tabs/auth/state/index.tsx

```diff
@@ -858,13 +914,30 @@ export class DraftProviderConfig extends Model({
     return this.oauthSecret.trim() === "" ? "Secret is required" : null;
   }
 
+  @computed
+  get webauthnRelyingOriginError() {
+    return this.webauthnRelyingOrigin.trim() === ""
+      ? "Relying origin is required"
+      : null;
+  }
```

Comment:
Do you think we ought to do a quick `new URL` check here to ensure it's a valid URL? We might even want to ensure that no pathname is set, etc.